_docs/tutorial/05-adding-a-new-version-of-a-microservice.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Adding a new version of a microservice
+overview: Overview
+
+order: 05
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# Deploy and release a new version of _reviews_
+
+Let's deploy a new version of the _reviews_ microservice, the one that will return the ratings provided by reviewers, as a number of stars, with the color of stars. In the real life, we would perform lint tests, unit tests, integration tests, end-to-end tests and tests in the staging environment.
+
+1. Deploy the new version of the _reviews_ microservice without the `app` label. Without the `app` label, our new version of the microservice will not be selected to provide the _reviews_ service. As such, it will not be called by the production code.
+  ```bash
+  kubectl apply -f bookinfo-reviews-v2-without-app-label.yaml
+  ```
+
+2. Let's access our application and see that the deployed microservice did not disrupt it. So far so good.
```

Comment:
curl command to do this?

Reply:
@geeknoid I do not think it is a good idea to access the productpage by curl. I do not see a reason why anyone would not use the browser. You just open the browser with the productpage once, and then hit refresh multiple times during the tutorial. This will show what happens graphically - black/red/no review stars, reviews unavailable etc. curl will just dump the html, without any clue what happened. During this tutorial I think it is important to see the productpage visually - to convince yourself that you can perform multiple transformations without disrupting the application.

Reply:
@geeknoid And this is again an example that this is a part of a continuous tutorial - you open the browser at the step 3 to access the productpage, and then you just hit refresh on the following steps.